api-surface-swift

```diff
@@ -1573,3 +1573,390 @@ public enum WebViewTracking
+public class Recorder: Recording
+    public struct Context: Equatable
+        public let privacy: SessionReplayPrivacyLevel
+    deinit
+public typealias NodeID = Int64
+public final class NodeIDGenerator
+    public func nodeID(view: UIView, nodeRecorder: SessionReplayNodeRecorder) -> NodeID
+public protocol SessionReplayNodeRecorder
+    func semantics(of view: UIView, with attributes: SessionReplayViewAttributes, in context: SessionReplayViewTreeRecordingContext) -> SessionReplayNodeSemantics?
+    var identifier: UUID
+public protocol SessionReplayNodeWireframesBuilder
+    var wireframeRect: CGRect
+    func buildWireframes(with builder: SessionReplayWireframesBuilder) -> [SRWireframe]
+public struct SessionReplayViewTreeRecordingContext
+    public let recorder: Recorder.Context
+    public let ids: NodeIDGenerator
+public struct SessionReplayNode
+    public let viewAttributes: SessionReplayViewAttributes
+    public let wireframesBuilder: SessionReplayNodeWireframesBuilder
+    public init(viewAttributes: SessionReplayViewAttributes, wireframesBuilder: SessionReplayNodeWireframesBuilder)
+public protocol SessionReplayResource
+    func calculateIdentifier() -> String
+    func calculateData() -> Data
+public struct SessionReplayViewAttributes: Equatable
+    public let frame: CGRect
+    public let backgroundColor: CGColor?
+    public let layerBorderColor: CGColor?
+    public let layerBorderWidth: CGFloat
+    public let layerCornerRadius: CGFloat
+    public let alpha: CGFloat
+public protocol SessionReplayNodeSemantics
+    static var importance: Int
+    var subtreeStrategy: SessionReplayNodeSubtreeStrategy
+    var nodes: [SessionReplayNode]
+    var resources: [SessionReplayResource]
+public enum SessionReplayNodeSubtreeStrategy
+    case record
+    case ignore
+public struct SessionReplayInvisibleElement: SessionReplayNodeSemantics
+    public static let importance: Int = 0
+    public let subtreeStrategy: SessionReplayNodeSubtreeStrategy
+    public let nodes: [SessionReplayNode] = []
+    public let resources: [SessionReplayResource] = []
+    public static let constant = SessionReplayInvisibleElement()
+public struct SessionReplaySpecificElement: SessionReplayNodeSemantics
+    public static let importance: Int = .max
+    public let subtreeStrategy: SessionReplayNodeSubtreeStrategy
+    public let nodes: [SessionReplayNode]
+    public let resources: [SessionReplayResource]
+    public init(subtreeStrategy: SessionReplayNodeSubtreeStrategy,nodes: [SessionReplayNode],resources: [SessionReplayResource] = [])
+public final class DDSessionReplay: NSObject
+    public static func enable(with configuration: DDSessionReplayConfiguration)
+public final class DDSessionReplayConfiguration: NSObject
+    @objc public var replaySampleRate: Float
+    @objc public var defaultPrivacyLevel: DDSessionReplayConfigurationPrivacyLevel
+    @objc public var customEndpoint: URL?
+    public required init(replaySampleRate: Float)
```

Comment:
@DataDog/rum-mobile-ios Since `DatadogSessionReplay` module contains ObjC bindings, they leak into `api-surface-swift` file. This needs to be addressed in generator.

Reply:
Yes indeed!